### PR TITLE
log tail show fields

### PR DESCRIPTION
**Pull request by javasoze**

### CODE REVIEW

File: tools/logtail.py [hunk 1]
```diff
@@ -14,15 +14,18 @@
...
                     help="time filter, e.g. May 20 2017 10:11:13,10 min ago")
 
 
-def doQuery(rpc, req):
+def doQuery(rpc, req, hideRaw):
     response = rpc.do_post("/v1/query", req)
     data = response.data
     result = simplejson.loads(data)
-    resultview.printResult(result)
+    print hideRaw
```
Comment (keitaf) [inline tools/logtail.py]: is this for debugging?

File: tools/logtail.py [hunk 2]
```diff
@@ -37,6 +40,9 @@ def main():
     else:
         table, url = args.addr.split("@", 1)
 
+    if args.field != None:
```
Comment (keitaf) [inline tools/logtail.py]: nit: I think you don't need `!= None` as None = false and not None = true in python

File: tools/resultview.py [hunk 3]
```diff
@@ -38,9 +38,12 @@ def rawPayload(hit):
     return raw.encode('utf-8').strip()
 
 
-def printResult(result):
+def printResult(result, fields, showRaw):
     hits = result["hits"]
-    printHits(None, hits, True)
+    if fields != None:
```
Comment (keitaf) [inline tools/resultview.py]: looks like you can just call `printHits(fields, hits, True, showRaw)`? if `fields` is None it'll be passed to printHits

File: tools/logtail.py [hunk 4]
```diff
@@ -14,15 +14,18 @@
 parser.add_argument("-q", "--query", type=str, default=None, help="query string")
 parser.add_argument("-s", "--stream", type=bool, default=False, help="streaming mode")
 parser.add_argument("-d", "--delay", type=int, default=5, help="delay in seconds in streaming mode")
+parser.add_argument("-f", "--field", type=str, default=None, help="default field value to display")
```
Comment (keitaf) [inline tools/logtail.py]: can we make `-f` option to support multiple fields? e.g., `-f level,logger,message`

Comment (javasoze) [reply tools/logtail.py]: yes, that would be better!

File: tools/logtail.py (hunk 1 above)
Comment (javasoze) [reply tools/logtail.py]: yup, removed.

File: tools/logtail.py (hunk 2 above)
Comment (javasoze) [reply tools/logtail.py]: got it!

File: tools/resultview.py (hunk 3 above)
Comment (javasoze) [reply tools/resultview.py]: yes, how embarassing :)

**Pull request merged**